portfolio/src/main/webapp/script.js

```diff
@@ -42,4 +42,18 @@ async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
```

Comment:
I must have missed this but this doesn't need an `await`.

`const quote = response.text();`

await is just used for code that can be executed (asynchronously). When you reference the variable (in this case, response) in another line, the code execution stops and waits until response is defined before assigning a value to quote.

Reply:
Actually nevermind, I retract this. I think I see some documentation saying you should keep await on dependent variables. Ignore what I said here.